### DISCUSSION TITLE
Any idea when is the hv-uikit-react going to support react 18?

### DISCUSSION THREAD

**Discussion by ferm12** [lumada-design/hv-uikit-react#3241] [General]
Any idea when is the hv-uikit-react going to support react 18? 
In our app we are using react 18. So do we need to downgrade to 17 in order to use the hv-uikit-react?

**Comment by zettca** [comment]
Hey @ferm12 👋 

Apologies for the delay. UI Kit v5 supports both React 17 and 18
